Review API

evirens opened the pull request:
Wrote API

Commit:
Review API
Wrote API

JohanTorfs reviewed (commented):
Gingen we de link niet via id doen ipv via naam?

Commit:
Piece Id
Reviews now only store the piece Id instead of name
Added get all reviews method at / mapping

JohanTorfs reviewed (approved):
Looks good!

Commit:
Get all pieces

JohanTorfs reviewed (approved):
Ok

Commit:
Literally just moved a method

JohanTorfs reviewed (approved):
I APPROVE!
